tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -260,66 +234,71 @@ def example_plot(ax, fontsize=12, hide_labels=False):
-# and the minimum distance between adjacent plots.  It is specified in
-# inches by the keyword arguments ``w_pad`` and ``h_pad`` to the function
-# `~.figure.Figure.set_constrained_layout_pads`:
+# Padding between axes is controlled in the horizontal by ``w_pad`` and
+# ``wspace``, and vertical by ``h_pad`` and ``hspace``.  These can be edited
+# via `~.figure.Figure.set_constrained_layout_pads`.  *w/h_pad* are
+# the minimum space around the axes in units of inches:
 
 fig, axs = plt.subplots(2, 2, constrained_layout=True)
 for ax in axs.flat:
     example_plot(ax, hide_labels=True)
-fig.set_constrained_layout_pads(w_pad=4/72, h_pad=4/72, hspace=0, wspace=0)
+fig.set_constrained_layout_pads(w_pad=4 / 72, h_pad=4 / 72, hspace=0, wspace=0)
+
+##########################################
+# Spacing between subplots is further set by ``wspace`` and ``hspace``. These
+# are specified as a fraction of the size of the subplot group as a whole.
+# If these values are smaller than w_pad or h_pad, then the fixed pads are used
+# instead. Note in the below how the space at the edges doesn't change from
+# the above, but the space between subplots does.
 
 fig, axs = plt.subplots(2, 2, constrained_layout=True)
 for ax in axs.flat:
     example_plot(ax, hide_labels=True)
-fig.set_constrained_layout_pads(w_pad=2/72, h_pad=2/72, hspace=0, wspace=0)
+fig.set_constrained_layout_pads(w_pad=4 / 72, h_pad=4 / 72, hspace=0.2,
+                                wspace=0.2)
 
 ##########################################
-# Spacing between subplots is set by ``wspace`` and ``hspace``. There are
-# specified as a fraction of the size of the subplot group as a whole.
-# If the size of the figure is changed, then these spaces change in
-# proportion.  Note in the below how the space at the edges doesn't change from
-# the above, but the space between subplots does.
+# If there are more than two columns, the wspace is shared between them,
+# so here the wspace is divided in 2, with a wspace of 0.1 between each column:
 
-fig, axs = plt.subplots(2, 2, constrained_layout=True)
+fig, axs = plt.subplots(2, 3, constrained_layout=True)
 for ax in axs.flat:
     example_plot(ax, hide_labels=True)
-fig.set_constrained_layout_pads(w_pad=2/72, h_pad=2/72, hspace=0.2, wspace=0.2)
-
+fig.set_constrained_layout_pads(w_pad=4 / 72, h_pad=4 / 72, hspace=0.2,
+                                wspace=0.2)
 
 ##########################################
-# Spacing with colorbars
-# -----------------------
-#
-# Colorbars will be placed ``wspace`` and ``hsapce`` apart from other
-# subplots. The padding between the colorbar and the axis it is
-# attached to will never be less than ``w_pad`` (for a vertical colorbar)
-# or ``h_pad`` (for a horizontal colorbar). Note the use of the ``pad`` kwarg
-# here in the ``colorbar`` call.  It defaults to 0.02 of the size
-# of the axis it is attached to.
+# GridSpecs also have optional ``hspace`` and ``wspace`` keyword arguments,
```

Comment:
Sorry, I should have caught the styling before:
```suggestion
# GridSpecs also have optional *hspace* and *wspace* keyword arguments,
```

Reply:
OK, I think I caught most of the double backticks in this file.